<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorLayerSaveAsDialogBase</class>
 <widget class="QDialog" name="QgsVectorLayerSaveAsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save vector layer as...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item row="7" column="0">
    <widget class="QLabel" name="mSymbologyExportLabel">
     <property name="text">
      <string>Symbology export</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="mSkipAttributeCreation">
     <property name="toolTip">
      <string>This allows one to surpress attribute creation as some OGR drivers (eg. DGN, DXF) don't support it.</string>
     </property>
     <property name="text">
      <string>Skip attribute creation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Format</string>
     </property>
     <property name="buddy">
      <cstring>mFormatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browseFilename">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QgsCollapsibleGroupBox" name="mOgrOptionsGroupBox">
     <property name="title">
      <string>Custom Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Data source</string>
        </property>
        <property name="buddy">
         <cstring>mOgrDatasourceOptions</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="mOgrDatasourceOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Layer</string>
        </property>
        <property name="buddy">
         <cstring>mOgrLayerOptions</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="mOgrLayerOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_6</zorder>
     <zorder>label_5</zorder>
     <zorder>mOgrDatasourceOptions</zorder>
     <zorder>mOgrLayerOptions</zorder>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QCheckBox" name="mAddToCanvas">
     <property name="text">
      <string>Add saved file to map</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="leCRS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CRS</string>
     </property>
     <property name="buddy">
      <cstring>leCRS</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Save as</string>
     </property>
     <property name="buddy">
      <cstring>leFilename</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="mEncodingComboBox"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="mScaleLabel">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="mFormatComboBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Encoding</string>
     </property>
     <property name="buddy">
      <cstring>mEncodingComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QSpinBox" name="mScaleSpinBox">
     <property name="prefix">
      <string>1:</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>50000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leFilename">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QgsCollapsibleGroupBox" name="mLayerOptionsGroupBox">
     <property name="title">
      <string>Layer Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="browseCRS">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="mCRSSelection"/>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QComboBox" name="mSymbologyExportComboBox"/>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QgsCollapsibleGroupBox" name="mDatasourceOptionsGroupBox">
     <property name="title">
      <string>Datasource Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>mDatasourceOptionsGroupBox</zorder>
  <zorder>mLayerOptionsGroupBox</zorder>
  <zorder>mFormatComboBox</zorder>
  <zorder>mScaleSpinBox</zorder>
  <zorder>mScaleLabel</zorder>
  <zorder>mAddToCanvas</zorder>
  <zorder>mSkipAttributeCreation</zorder>
  <zorder>leFilename</zorder>
  <zorder>browseCRS</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>mEncodingComboBox</zorder>
  <zorder>mSymbologyExportLabel</zorder>
  <zorder>label</zorder>
  <zorder>mSymbologyExportComboBox</zorder>
  <zorder>browseFilename</zorder>
  <zorder>leCRS</zorder>
  <zorder>mCRSSelection</zorder>
  <zorder>label_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>mOgrOptionsGroupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mFormatComboBox</tabstop>
  <tabstop>leFilename</tabstop>
  <tabstop>browseFilename</tabstop>
  <tabstop>mEncodingComboBox</tabstop>
  <tabstop>leCRS</tabstop>
  <tabstop>browseCRS</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsVectorLayerSaveAsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsVectorLayerSaveAsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
